communication.rst

```diff
@@ -52,12 +52,12 @@ General Python questions should go to `python-list`_ or `tutor`_
 or similar resources, such as StackOverflow_ or the ``#python`` IRC channel
 on Freenode_.
 
-`Core-Workflow <https://mail.python.org/mailman/listinfo/core-workflow>`_
+`Core-Workflow <https://mail.python.org/mm3/mailman3/lists/core-workflow.python.org/>`_
 mailing list is the place to discuss and work on improvements to the CPython
 core development workflow.
 
 A complete list of Python mailing lists can be found at https://mail.python.org.
-Most lists are also mirrored at http://news.gmane.org/ and can be read and
+Most lists are also mirrored at [GMANE](http://gmane.org) and can be read and
```

Comment:
Sorry @vaibssingh. I gave you the markdown link format instead of reStructuredText. This link will need to be changed to use rst. Thanks.

Reply:
@willingc I am sorry but I don't understand :sweat_smile: Can you please provide an example?

Reply:
In Markdown markup, a link is specified like this: `[GMANE](http://gmane.org)`

In reStructuredText markup, a link is specified like this: 
```
`GMANE <http://gmane.org>`_
```

Reply:
Thank you for your clear explanation! Shall keep in mind for next time.

Reply:
Done!